templates.json

```diff
@@ -702,9 +690,34 @@
         "path": "editor-extension-collection"
       }
     ],
-    "organizationBetaFlags": [
-      "editor_extension_collection"
-    ]
+    "organizationBetaFlags": ["editor_extension_collection"]
+  },
+  {
+    "identifier": "discount_details_ui",
+    "name": "Discount Details - UI Extension",
+    "defaultName": "discount-details-ui",
+    "group": "Admin",
+    // TODO is this correct?
+    "supportLinks": ["https://shopify.dev/docs/apps/discounts"],
+    // TODO is this correct? It is a private repo currently
+    "url": "https://github.com/Shopify/discount-extensions-examples",
+    "type": "ui_extension",
+    "extensionPoints": [],
+    // Double check these
+    "supportedFlavors": [
+      {
+        "name": "JavaScript React",
+        "value": "react",
+        "path": "discount-details"
+      },
+      {
+        "name": "TypeScript React",
+        "value": "typescript-react",
+        "path": "discount-details"
+      }
+    ],
+    // organization or app beta flags?
+    "organizationBetaFlags": ["extensions_only_discounts_app"]
```

Comment:
We should look up `order_routing_extensibility_partner` to determine how that is configured, but it looks to be an organization beta not an app beta?